### ISSUE TITLE
4.12.0 Regression: CreateSnapshot causes images to be loaded many times

### ISSUE THREAD

**Issue by gcoombe** [cypress-io/cypress#8679]
### Current behavior
Currently if `log` is called (internally within a cypress command or by `cy.log`) all images which are in the process of loading will reload. On some pages this is generating so many network calls that xhr requests are getting stalled to the point that all tests time out.

I think this occurs when we clone the body element here https://github.com/cypress-io/cypress/blob/develop/packages/driver/src/cy/snapshots.js#L155 

On some internal pages we have many XHR requests (which generate many logs) and many larger images. The reloading behavior is to the point that our tests time out due to requires xhr/fetch calls getting stuck behind all the images.

### Desired behavior
Calling cy.log shouldn't trigger a reloading of all images

### Test code to reproduce
Use this snippet with cy.open and disable cache off:

```js
    cy.visit('https://github.com/');
    Array(15).fill().forEach(() => {
      cy.log('Test log')
      cy.wait(100);
    })
```

Using the above there were 1762 image requests when loading github homepage. Each image was loaded 33 times
![](https://user-images.githubusercontent.com/764083/94351677-7eb1f280-0010-11eb-9fd4-2e73c0857aa1.png)
![](https://user-images.githubusercontent.com/764083/94351678-82457980-0010-11eb-9691-08516d1d0c97.png)

### Versions
* Cypress 5.2
* Using electron
* cypress open

**Comment by mtc3bmtc3b** [comment]
We are observing the same issue, also with Cypress 5.2 using cypress open, but using Chrome.

**Comment by jennifer-shehane** [comment]
This was introduced in 4.12.0, likely by #8080, which you referenced at the line of code https://github.com/cypress-io/cypress/blob/develop/packages/driver/src/cy/snapshots.js#L155

The waiting of these resources could significantly slow down whatever is waiting on these resources to load.

### Reproducible example

**Disable cache in the Network tab of DevTools**

`index.html`

```html
<html>
<body>
  <img src="https://upload.wikimedia.org/wikipedia/commons/thumb/1/18/City_Lights_of_the_United_States_2012.jpg/2560px-City_Lights_of_the_United_States_2012.jpg">
</body>
</html>
```

spec.js

```js
it('test', () => {
  cy.visit('index.html')
  Array(15).fill().forEach(() => {
    cy.log('Test log')
  })
})
```

### 4.11.0

<img width="1060" alt="Screen Shot 2020-09-29 at 8 41 14 AM" src="https://user-images.githubusercontent.com/1271364/94504778-8f8c7f00-022f-11eb-872a-94a81b60a7c8.png">


### 4.12.0

<img width="1059" alt="Screen Shot 2020-09-29 at 8 38 51 AM" src="https://user-images.githubusercontent.com/1271364/94504640-42a8a880-022f-11eb-9d86-739a0fe51f9b.png">

**Comment by crfrolik** [comment]
I don't know what "CreateSnapshot" is but I'm seeing the same images fly by the console repeatedly while a test runs, e.g.

```
GET /assets/adt/loading.gif 304 4.127 ms - -
```

(and many more images, repeated)

Could this be related?

**Comment by crfrolik** [comment]
(Chrome DevTools network view: ah, is this what is meant by "CreateSnapshot"?)

![image](https://user-images.githubusercontent.com/60711200/94744690-67903f00-0347-11eb-9286-67847ee3154b.png)

**Comment by jennifer-shehane** [comment]
Snapshots are what we call the DOM that's captured on each command, so that when you scroll over the command log the DOM is rerendered to the state it was previously (aka time travel feature).

@crfrolik See if the issue of the requests is resolved after downgrading to 4.11.0, that should narrow down if this is the same issue.

**Comment by crfrolik** [comment]
Thank you @jennifer-shehane - I downgraded to cypress 4.11.0 and the issue does not occur there, so that confirms my problem is a duplicate of this one.

**Comment by gcoombe** [comment]
As a side note another issue this causes on our end is that cy.visit ends up timing out as images constantly load and the `Load` browser event never fires as a result

**Comment by hernqvistdaniel** [comment]
@jennifer-shehane I hope this gets resolved soon, we are running recursive forms of 60-70 inputs (medtech). I'm only clicking inputs and typing. After half a test the time scales up so much, running 4-5 of these forms takes 30 minutes (imagine the time writing each test).
Atleast it works on 4.11. Soon I will not be able to justify cypress for my CTO.
Let me know if I can be of any help, much love!

**Comment by hernqvistdaniel** [comment]
Yes! Perfect @chrisbreiding !
